relevanceai/_api/endpoints/services/centroids.py

```diff
@@ -120,7 +80,7 @@ def insert(
             },
         )
 
-    def documents(
+    def list(
```

Comment:
You will need to add back a `documents` function. Just add this 1 line: 

```
documents = list
```

Reply:
but the end point is called ListCentroids`

Reply:
`list` and `documents` methods called the same endpoint anyway. Are you saying you want the function called `documents` instead?